crates/l2/Makefile

```diff
@@ -172,11 +174,21 @@ ci_test: ## 🚧 Runs the L2's integration test, used by the github's CI
 	# We create an empty .env file simply because if the file
 	# does not exists, the container fails to write to it.
 	touch .env
+	# Integration tests
 	CI_ETHREX_WORKDIR=${CI_ETHREX_WORKDIR} docker compose -f ${ethrex_L2_DOCKER_COMPOSE_PATH} down
 	CI_ETHREX_WORKDIR=${CI_ETHREX_WORKDIR} docker compose -f ${ethrex_L2_DOCKER_COMPOSE_PATH} up -d --build
 	cargo test l2 --release -- --nocapture --test-threads=1
 	CI_ETHREX_WORKDIR=${CI_ETHREX_WORKDIR} docker compose -f ${ethrex_L2_DOCKER_COMPOSE_PATH} down
 
+	# State reconstruction tests
+	rm -rf $$PWD/store
```

Comment:
Everything else is using relative paths and this is equivalent to `./store` AFAICT. Why not just use that?

Reply:
It's for consistency with the following line, that requires a full path:
```
make init-l2-no-metrics ethrex_L2_DEV_LIBMDBX=$$PWD/store 2>&1 >/dev/null &
```
In the case you mentioned it's not really needed. If you prefer I can remove it